assets/styles/components/_header.scss

```diff
@@ -288,8 +288,7 @@
     height: 100%;
 
     span {
-      font-weight: lighter;
```

Comment:
this part felt a bit strange for what it was doing (the `/m/` in the header when viewing a magazine) `lighter` may or may not work in chromium, seems to but it all depends on the font I think, if there is no `lighter` version of the font it can't do anything, and the `font-size` was offsetting the `/m/` in other fonts (making it raised above like an exponent or something). I'll double check around but feel like this sort of preserves the way it looked for me, with other fonts